Sources/BraveWallet/Crypto/Asset Details/NFTDetailView.swift

```diff
@@ -127,3 +140,9 @@ struct NFTDetailView: View {
     .navigationBarTitle(Strings.Wallet.nftDetailTitle)
   }
 }
+
+private extension String {
+  var isSVGImage: Bool {
+    hasPrefix("data:image/svg") || hasSuffix(".svg")
+  }
```

Comment:
I think missed using this in the view's `isSVGImage` computed property

Reply:
its actually not needed. removed in [a373fc4](https://github.com/brave/brave-ios/pull/6555/commits/a373fc41f76ce4d0cc033555906ab72ad4f631c3)